superset/thumbnails/digest.py

```diff
@@ -92,11 +93,15 @@ def _adjust_string_with_rls(
 def get_dashboard_digest(dashboard: Dashboard) -> str:
     config = current_app.config
     datasources = dashboard.datasources
-    executor_type, executor = get_executor(
-        executor_types=config["THUMBNAIL_EXECUTE_AS"],
-        model=dashboard,
-        current_user=get_current_user(),
-    )
+    try:
+        executor_type, executor = get_executor(
+            executors=config["THUMBNAIL_EXECUTORS"],
+            model=dashboard,
+            current_user=get_current_user(),
+        )
+    except ExecutorNotFoundError:
+        return ""
```

Comment:
Maybe `None` will be a better representation? If you agree, please change all digest generation methods.